datamodel/extra_scripts/oracle/107_brk_views.sql

```diff
@@ -1,416 +1,1417 @@
-      ELSE nnp2.NAAM
-    END AS app_re_zak_recht_naam,
---    nnp2.NAAM as r_nnp,
-ar.SC_KAD_IDENTIF as app_re_identificatie,
- to_number(ar.KA_APPARTEMENTSINDEX) as appartementsindex --,
--- ar.FK1_NIET_NAT_PERSOON as app_re_vve,
--- ar_vve_nnp.naam as app_re_vve_naam,
--- ar_vve_innp.rechtsvorm as app_re_vve_rechtsvorm,
--- ar_vve_innp.rsin as app_re_vve_rsin
-from v_kad_perceel_eenvoudig kpe
-join zak_recht zr on (zr.FK_7KOZ_KAD_IDENTIF = kpe.SC_KAD_IDENTIF)
-  left join nat_prs np1 on (np1.SC_IDENTIF = zr.FK_8PES_SC_IDENTIF)
-  left join ingeschr_nat_prs inp1 on (inp1.SC_IDENTIF = np1.SC_IDENTIF)
-  left join niet_nat_prs nnp1 on (nnp1.sc_identif = zr.FK_8PES_SC_IDENTIF)
-  left join ingeschr_niet_nat_prs innp1 on (innp1.sc_identif = nnp1.sc_identif)
-join brondocument bd1 on (bd1.tabel = 'ZAK_RECHT' and bd1.tabel_identificatie = zr.kadaster_identif)
-join brondocument bd2 on (bd2.tabel = 'ZAK_RECHT' and bd2.tabel_identificatie <> zr.kadaster_identif and bd2.identificatie = bd1.identificatie)
-join zak_recht zr2 on (zr2.kadaster_identif = bd2.tabel_identificatie)
-  left join nat_prs np2 on (np2.SC_IDENTIF = zr2.FK_8PES_SC_IDENTIF)
-  left join ingeschr_nat_prs inp2 on (inp2.SC_IDENTIF = np2.SC_IDENTIF)
-  left join niet_nat_prs nnp2 on (nnp2.sc_identif = zr2.FK_8PES_SC_IDENTIF)
-  left join ingeschr_niet_nat_prs innp2 on (innp2.sc_identif = nnp2.sc_identif)
-join app_re ar on (ar.SC_KAD_IDENTIF = zr2.FK_7KOZ_KAD_IDENTIF)
-join niet_nat_prs ar_vve_nnp on (ar_vve_nnp.sc_identif = ar.FK_2NNP_SC_IDENTIF)
-join INGESCHR_NIET_NAT_PRS ar_vve_innp on (ar_vve_innp.sc_identif = ar_vve_nnp.sc_identif)
-where bd1.omschrijving like 'betrokkenBij%'
-and zr2.FK_8PES_SC_IDENTIF is not null
-order by kpe.SC_KAD_IDENTIF, kpe.straat, kpe.huisnummer, kpe.toevoeging, kpe.huisletter,  to_number(KA_APPARTEMENTSINDEX);
-
--- percelen plus appartementen op de percelen
-CREATE OR REPLACE VIEW v_bd_app_re_and_kad_perceel
-                                 AS
-  SELECT CAST(ROWNUM AS INTEGER) AS objectid,
-    qry.*
-  FROM
-    (SELECT p.sc_kad_identif AS kadaster_identificatie,
-      'perceel'              AS type,
-      p.ka_deelperceelnummer,
-      '' AS ka_appartementsindex,
-      p.ka_perceelnummer,
-      p.ka_kad_gemeentecode,
-      p.ka_sectie,
-      p.begrenzing_perceel
-    FROM kad_perceel p
-    UNION ALL
-    SELECT ar.sc_kad_identif AS kadaster_identificatie,
-      'appartement'          AS type,
-      ''                     AS ka_deelperceelnummer,
-      ar.ka_appartementsindex,
-      ar.ka_perceelnummer,
-      ar.ka_kad_gemeentecode,
-      ar.ka_sectie,
-      kp.begrenzing_perceel
-    FROM v_bd_app_re_all_kad_perceel v
-    JOIN kad_perceel kp
-    ON v.perceel_identif = kp.sc_kad_identif
-    JOIN app_re ar
-    ON v.app_re_identif = ar.sc_kad_identif
-    ) qry ;
-
--- aankoopdatum uit brondocumenten
+/*
+Views for visualizing the BRK data.
+versie 2
+26-6-2018
+*/
+--drop view v_subject cascade;
```

Comment:
dit is geen geldige oracle sql, in 11.2 is er wel drop view <view> cascade constraints